test-cases/performance/perf-regression.100threads.30M-keys-i4i-enterprise.yaml

```diff
@@ -32,3 +32,5 @@ email_recipients: ['scylla-perf-results@scylladb.com']
 
 custom_es_index: 'performancestatsv2'
 use_hdr_cs_histogram: true
+
+append_scylla_args: '--blocked-reactor-notify-ms 5'
```

Comment:
Not good.
Add the exact same row we have for all other performance tests.

Reply:
@roydahan , Done